admin_manual/release_notes.rst

```diff
@@ -16,6 +17,105 @@ Release Notes
 * :ref:`8.0_release_notes_label`
 * :ref:`7.0_release_notes_label`
 
+.. _10.0.8_release_notes_label:
+
+Changes in 10.0.8
+-----------------
+
+Dear ownCloud administrator, please find below the changes and known issues in ownCloud Server 10.0.8 that need your attention. You can also read `the full ownCloud Server changelog`_ for further details on what has changed.
+
+PHP 5.6 deprecation
+~~~~~~~~~~~~~~~~~~~
+PHP 5.6 active support has ended on 19 Jan 2017 and security support `will be dropped by the end of 2018 <https://secure.php.net/supported-versions.php>`_. Many libraries used by ownCloud (including the QA-Suite *PHPUnit*) will therefore not be maintained actively anymore which forces ownCloud to drop support in one of the next minor server versions as well. Please make sure to upgrade to PHP 7 as soon as possible. See the `system requirements in the ownCloud documentation <https://doc.owncloud.com/server/latest/admin_manual/installation/system_requirements.html#officially-recommended-supported-options>`_.
```

Comment:
We say to upgrade to PHP 7. That could mean PHP7.0 or 7.1
Because of ownCloud 10.1 beeing not that far away and that release needs mandatory PHP 7.1, 
I would propose a recommandation to upgrade to PHP 7.1.

Reply:
And PHP 7.0 security support ends 3 Dec 2018 - actually a few weeks before 5.6 security support on 31 Dec 2018 - http://php.net/supported-versions.php

So it is silly for anyone to upgrade just to 7.0

Reply:
Good point, adjusted that.